typescript/hx-adl/gen-adl.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I'll remove this when I mark the PR ready for review, for now, I'll leave it in so that you can see what commands lead to the generated output